jekyll/_cci2/orb-intro.md

```diff
@@ -12,40 +12,103 @@ contentTags:
   - Server v3.x
 ---
 
-* TOC
-{:toc}
+Use orbs to:
```

Comment:
Would this list not go under the introduction section? It seems odd to have an intro before an intro.

Reply:
https://circleci.atlassian.net/wiki/spaces/PEX/pages/6679953646/WIP+Content+templates#Conceptual we added a section pre-intro to highlight use cases after feedback that people want to know before reading whether content is relevant for them. Happy to talk this over though!